README.md

```diff
@@ -1,15 +1,26 @@
-# StructuredLogger
+# Logr
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/structured_logger`. To experiment with that code, run `bin/console` for an interactive prompt.
+Structured logging with metrics and events.
 
-TODO: Delete this and the text above, and describe your gem
+![Travis](https://api.travis-ci.org/sspinc/avro2kafka.svg?branch=master)
```

Comment:
The project should be built with CircleCI and the README should include a badge from CircleCI. Remove if there is no CI yet.